contrib/lib/src/space_helmet/helmet.rs

```diff
@@ -0,0 +1,131 @@
+use std::sync::atomic::{AtomicBool, Ordering};
+
+extern crate rocket;
+use rocket::fairing::{Fairing, Info, Kind};
+use rocket::http::Header;
+use rocket::Rocket;
+use rocket::{Request, Response};
+
+use space_helmet::policy::*;
+
+pub struct Helmet<'a> {
+    expect_ct_policy: Option<ExpectCTPolicy<'a>>,
+    no_sniff_policy: Option<NoSniffPolicy>,
+    xss_protect_policy: Option<XSSPolicy<'a>>,
+    frameguard_policy: Option<FramePolicy<'a>>,
+    hsts_policy: Option<HSTSPolicy>,
```

Comment:
It's pretty weird that there are _three_ fields that control HSTS. Can we simplify this? I can't quite see why only one field wouldn't work.

Reply:
I'm open to recommendations.
each has its own purpose.
hsts_policy is whatever policy the user sets or the default.
force_hsts has to be an atomic so we can use it from on_launch.
force_hsts_policy says what policy to force, the reason I added this is that its
maybe a little bit clearer than just burying it down in apply -- but mostly because it makes things work out with try_apply_header which takes an identifier as opposed to a header.

Again if you have a reccomendation I'm open to it, but I'm good with it as it is. I think its 
pretty simple and understandable.